### ISSUE TITLE
Cannot find module 'node_modules/replicate/lib/errors.js' 

### ISSUE THREAD

**Issue by zeke** [replicate/replicate-javascript#24]
Seeing this error in the server logs from https://github.com/replicate/scribble-diffusion/pull/46

I thought this was maybe a `files` thing in `package.json`, but the file is there: https://unpkg.com/browse/replicate@0.5.0/lib/

```
2023-03-13T19:06:25.472Z  2023-03-13T19:06:26.632Z	4c7e6aa0-c7c4-4f91-b02c-e0f56b9f7e38	ERROR	Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/var/task/node_modules/replicate/lib/errors.js' imported from /var/task/node_modules/replicate/lib/ReplicateClient.js
2023-03-13T19:06:25.472Z      at new NodeError (node:internal/errors:400:5)
2023-03-13T19:06:25.472Z      at finalizeResolution (node:internal/modules/esm/resolve:331:11)
2023-03-13T19:06:25.472Z      at moduleResolve (node:internal/modules/esm/resolve:994:10)
2023-03-13T19:06:25.472Z      at moduleResolveWithNodePath (node:internal/modules/esm/resolve:938:12)
2023-03-13T19:06:25.472Z      at defaultResolve (node:internal/modules/esm/resolve:1202:79)
2023-03-13T19:06:25.472Z      at nextResolve (node:internal/modules/esm/loader:163:28)
2023-03-13T19:06:25.472Z      at ESMLoader.resolve (node:internal/modules/esm/loader:842:30)
2023-03-13T19:06:25.472Z      at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
2023-03-13T19:06:25.472Z      at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
2023-03-13T19:06:25.472Z      at link (node:internal/modules/esm/module_job:76:36) {
2023-03-13T19:06:25.472Z    code: 'ERR_MODULE_NOT_FOUND'
2023-03-13T19:06:25.472Z  }
```

**Comment by zeke** [comment]
cc @erbridge any idea what's up here?

**Comment by zeke** [comment]
Is there any chance it's a caching issue where some filenames have changed case?

**Comment by erbridge** [comment]
> Is there any chance it's a caching issue where some filenames have changed case?

I doubt it for two reasons: `errors.js` was always lowercase, and Vercel is almost certainly running a case-sensitive filesystem.

**Comment by gr2m** [comment]
How can I replicate the above error?

**Comment by zeke** [comment]
1. Clone this PR: https://github.com/replicate/scribble-diffusion/pull/46
2. Run `npx vercel` to push your own copy

**Comment by zeke** [comment]
Or do something like this from scratch:

```
npx create-next-app my-app
cd my app
npm install replicate
// write some code
vercel
```

**Comment by gr2m** [comment]
Can you reproduce the problem with just `npm run build`? I'm not able to reproduce, not locally, not when deployed to production. Maybe let's pair on it?

**Comment by zeke** [comment]
Paired on this with @gr2m for a bit but didn't get anywhere.

Opened an issue upstream here: https://github.com/vercel/next.js/issues/47380

**Comment by zeke** [comment]
Resolved by https://github.com/replicate/replicate-javascript/pull/25#issuecomment-1481851418

**Comment by gr2m** [comment]
> Resolved by [#25 (comment)](https://github.com/replicate/replicate-javascript/pull/25#issuecomment-1481851418)

can you elaborate on what the problem was / how it was resolved?

**Comment by zeke** [comment]
Apparently it was the import assertion, as you suspected: https://github.com/vercel/next.js/issues/47380#issuecomment-1479691564 -- you and I tried removing that but we must not have tested it properly.

Anyway that code path doesn't exist any more since the client code was completely rewritten in https://github.com/replicate/replicate-javascript/pull/25
